root/static/scripts/common/i18n/localizeArtistRoles.js

```diff
@@ -0,0 +1,21 @@
+/*
+ * @flow
+ * Copyright (C) 2019 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+
+import {l_relationships} from './relationships';
+
+function localizeArtistRoles(roles: $ReadOnlyArray<string>) {
+  const translatedRoles = [];
+  roles.map(role => {
+    translatedRoles.push(l_relationships(role));
+  });
+  return translatedRoles;
+}
```

Comment:
You can simplify this function to just:
```JavaScript
function localizeArtistRoles(roles: $ReadOnlyArray<string>) {
  return roles.map(role => l_relationships(role));
}
```

(Map returns a new list with each item mapped through the given function - you're using it more like `forEach` which isn't necessary.)